faucet.py

```diff
@@ -52,7 +52,8 @@ def rest_faucet(recipient_address):
     """top up an account"""
     # recipient_address = request.form.get("account")
     # validate the address
-    logging.info("top off request for recipient_address")
+    logging.info("top up request for recipient_address")
+    print (logging.info)
```

Comment:
this is wrong. `info` is a function of the module `logging`, not a property. The correct version of this logging print would be: `logging.info(f"Top up request for {recipient_address}")`

Reply:
Got it updating it now.